app/utils/graph-tools.js

```diff
@@ -185,10 +196,12 @@ const hasProcessedDest = (graph, name) => {
  * @param  {Object}      inputGraph A directed graph representation { nodes: [], edges: [] }
  * @param  {Array|DS.PromiseArray|DS.PromiseManyArray}  [builds]     A list of build metadata
  * @param  {Array|DS.PromiseArray|DS.PromiseManyArray}  [jobs]       A list of job metadata
- * @param  {String}      [start]    Node name that indicates what started the graph
+ * @param  {String}   start     Node name that indicates what started the graph
+ * @param  {Boolean}  chainPR   Boolean flag for the chainPR setting
+ * @param  {number}   prNum     The pull request number
```

Comment:
```suggestion
 * @param  {String}   [start]     Node name that indicates what started the graph
 * @param  {Boolean}  [chainPR]   Boolean flag for the chainPR setting
 * @param  {number}   [prNum]     The pull request number
```
`[]` indicates optional or not required